programs/uxd/src/instructions/credix_lp/mint_with_credix_lp_depository.rs

```diff
@@ -219,7 +219,11 @@ pub(crate) fn handler(
     );
     require!(
         redeemable_amount_after_fees > 0,
-        UxdError::MinimumRedeemedCollateralAmountError
+        UxdError::MinimumMintedRedeemableAmountError
+    );
+    require!(
+        redeemable_amount_after_fees <= collateral_amount,
```

Comment:
nit: could simplify the checking as `(1..=collateral_amount).contains(&redeemable_amount_after_fees)` since they  return the same error code

Reply:
I think they do return a different error code right?

Reply:
I could merge both error codes if needed, but its a bigger change, so depends on you guys 👌

Reply:
oops sorry, they are different, ignore it pls 🤣